src/frontend/apps/lti_site/components/Warnings/index.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { render } from '@lib-tests/render';
+import { screen } from '@testing-library/react';
+
+import { Warnings } from './index';
+
+describe('<Warnings />', () => {
+  it('shows warnings', () => {
+    const warnings = ['warning1', 'warning2'];
+    render(<Warnings warnings={warnings} />);
+    expect(screen.getByText('warning1')).toBeInTheDocument();
+    expect(screen.getByText('warning2')).toBeInTheDocument();
+  });
+
+  it('does not show warnings', () => {
+    render(<Warnings warnings={undefined} />);
+    expect(screen.queryByText('warning1')).not.toBeInTheDocument();
+    expect(screen.queryByText('warning2')).not.toBeInTheDocument();
```

Comment:
Maybe more clean? WDYT
`render` must be from `@testing-library/react` though.

```suggestion
    const { container } = render(<Warnings warnings={undefined} />);
    expect(container).toBeEmptyDOMElement();
```

Reply:
I like it :+1: